aura/data_apis/util/type_definitions.py

```diff
@@ -0,0 +1,15 @@
+import base64
+from graphql import parse
+
+
+def read_and_encode_type_definitions(type_definitions: str):
+    try:
+        type_definitions_file = open(type_definitions)
+    except FileNotFoundError:
+        print("Could not open provided type definitions as file, trying as GraphQL SDL")
+    else:
+        type_definitions = type_definitions_file.read()
+
+    parse(type_definitions)
```

Comment:
Do we not need to handle exceptions from invalid type definitions/encoding to base64? Is it easy for users to understand the errors from these functions?